BKHOFF/iocBoot/iocBKHOFF-IOC-01/st-common.cmd

```diff
@@ -3,8 +3,8 @@
 < $(IOCSTARTUP)/init.cmd
 
 ## Configure devices
-#drvAsynIPPortConfigure("MC_CPU1","127.0.0.1:5024",0,0,0)
-drvAsynIPPortConfigure("MC_CPU1","192.168.1.67:200",0,0,0)
+drvAsynIPPortConfigure("MC_CPU1","$HOST",0,0,0)
```

Comment:
Shouldn't this be `$(HOST)`. As I recall, both are valid but the brackets are standard for IBEX